### ISSUE TITLE
Problem with urllib.parse

### ISSUE THREAD

**Issue by jhoelzl** [icoxfog417/pyoxford#3]
Hello,

i am using Pyhton 2.7.6 in an virtualenv. I have installed this module using `pip install pyoxford`.

When i run the python script for speech synthesis, i always get back this error:

> No module named parse

I found this in file speech_api.py

`import urllib.parse`

I have installed urllib3, is it possible to use that library for this?

`pip install urllib` does return:

>  Could not find a version that satisfies the requirement urllib

Or any other ideas? Thanks!

**Comment by jhoelzl** [comment]
On http://python3porting.com/noconv.html, there is an example how to use modules urllib / urlparse for Python 2 and urllib.parse for Python 3 in your code:

```
try:
    from urllib.request import urlopen
    from urllib.parse import urlparse
except ImportError:
    from urlparse import urlparse
    from urllib import urlopen

url = 'http://docs.python.org/library/'
parts = urlparse(url)
parts = parts._replace(path='/3.0'+parts.path)
page = urlopen(parts.geturl())
```

 Would be very nice if you can integreate this into the file  [pyoxford/pyoxford/speech_api.py](https://github.com/icoxfog417/pyoxford/blob/master/pyoxford/speech_api.py).

Thanks!

**Comment by icoxfog417** [comment]
Thank you for using the library, and nice suggest of implementation!

I have no plan to support Python2, but pull request for compatible implementation is welcome.

**Comment by jhoelzl** [comment]
Okay, I made a pull request: #4

**Comment by icoxfog417** [comment]
Fixed in 02194343e671a06da8137ca671c7abb4deb8b0fe
